web/src/message_edit.js

```diff
@@ -1280,13 +1284,33 @@ function handle_message_move_failure_due_to_time_limit(xhr, handle_confirm, on_h
     });
 }
 
+function show_message_moved_toast(toast_params) {
+    const new_stream_name = sub_store.maybe_get_stream_name(toast_params.new_stream_id);
+    const stream_topic = `#**${new_stream_name}>${toast_params.new_topic_name}**`;
+    const new_location_url = hash_util.by_stream_topic_url(
+        toast_params.new_stream_id,
+        toast_params.new_topic_name,
+    );
+    feedback_widget.show({
+        populate($container) {
+            const rendered_msg = render_message_moved_widget_body({
+                stream_topic,
+                new_location_url,
+            });
+            $container.html(rendered_msg);
```

Comment:
I'm surprised this doesn't trigger a linter error, but you should generally name a variable containing HTML `widget_body_html` or something, not `rendered_msg`.

Reply:
Understood